_source/providenceConfiguration/mainConfiguration/media_processing_media_type.csv

```diff
@@ -0,0 +1,32 @@
+Key,Description,Example
+QUEUE,Queue to deliver the media type to,"``QUEUE = mediaproc``"
```

Comment:
They all queue to mediaproc, as that's the only media queue processing plugin that was ever developed (we had expected we'd do more than one). We can potentially get rid of this, or at least make it default to mediaproc. At the least, we should note that only "mediaproc" is supported currently.